helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -472,11 +472,15 @@ public void updateDeploymentGroupHosts(final String name, final List<String> hos
           }
         }
 
+        log.info("starting zookeeper transaction for updateDeploymentGroupHosts on "
```

Comment:
It's no biggie, but I could see it being confusing if you the knowledge that these are expected to fail every once in a while.